external-crates/move/crates/move-core-types/src/language_storage.rs

```diff
@@ -133,13 +132,13 @@ impl TypeTag {
     }
 }
 
-impl FromStr for TypeTag {
-    type Err = anyhow::Error;
-
-    fn from_str(s: &str) -> Result<Self, Self::Err> {
-        parse_type_tag(s)
-    }
-}
+// impl FromStr for TypeTag {
+//     type Err = anyhow::Error;
+//
+//     fn from_str(s: &str) -> Result<Self, Self::Err> {
+//         parse_type_tag(s)
+//     }
+// }
```

Comment:
So i'm not sure how i feel able this. It seems an easy thing to do to just have the correct parser be in this crate so that we can have a FromStr impl that round-trips with its Display impl so i wonder what your rationale removing these